### ISSUE TITLE
alt attribute challenge ('Add Images to your Website') fails if alt contains hyphen 

### ISSUE THREAD

**Issue by davvwar** [freeCodeCamp/freeCodeCamp#13116]
On the first HTML challenge that involves images, you will fail the challenge if your alt text contains a hyphen.

This passes:  alt="Close up of kitten looking at camera".
This fails:  alt="Close-up of kitten looking at camera"> 

This failure state is not required. 

It's good practice to write alt attributes as they would be otherwise written. This includes hyphens. 

NVDA and JAWS parse them fine. NVDA announces "Close-up of kitten looking at camera" as "Close up of kitten looking at camera" -- i.e. it ignores the hyphen. 

That said, the alt attributes that FCC suggest include a full-stop and then a space. For example: alt="Close up of kitten looking at camera. ">. This is not something that the WCAG suggests. See: https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/G94. It seems like the alt attributes that FCC demands (i.e. no hyphens) and suggests (end with a period and then a space) are a bit quirky
 
* Chrome  (55.0.2883.87) 
* Operating System (Windows 10)
* Desktop

**Comment by dhcodes** [comment]
@davvwar thanks for the issue. This is fixed on the beta version of the site so when the new challenges are loaded to the main site, this will be fixed.
